lib/app.js

```diff
@@ -23,7 +23,7 @@ var defaultSettings = {
 
 function getPrompt(argv, schema, onComplete, index, results) {
   if (!Array.isArray(schema)) {
-    console.log(chalk.red('An error occurred'));
+    console.log(chalk.red('getPrompt schema must be an Array.'));
```

Comment:
This isn't useful to the average developer running `firebase-tools`, in fact it's pretty confusing. This repo needs a lot of work on messaging, but I don't think this is at all helpful here